File: backend/ServidorCanciones/src/main/java/com/servidorcanciones/Excepciones.java
package com.servidorcanciones;

/**
 *
 * @author IS Vega
 */
public class Excepciones {
    
    private Excepciones(){
        
    }

    public static final String CONEXIONBDCAIDA = "Error al conectar con la base de datos de SiReMu.";
    
    public static final String ERRORINTERNO = "Error interno del servidor de SiReMu.";
    
    public static final String ALBUMREPETIDO = "El usuario ya tiene un album registrado con ese nombre.";
    
    public static final String CANCIONREPETIDA = "Ya existe una canción con ese nombre en el album.";
    
    public static final String CANCIONNOENCONTRADA = "La canción solicitada no se encuentra registrada en la base de datos de SiReMu.";
    
    public static final String DATOSCANCIONFALTANTES = "No se recibieron los datos de la canción a subir.";
    
    public static final String NUMCANCIONESPRIVADASEXCEDIDO = "El usuario ya ha alcanzado el límite de 250 canciones privadas subidas al servidor.";
    
    public static final String NOHAYCANCIONES = "No hay canciones en esta lista de reproduccion";
}

File: backend2/ServidorListas/lib/excepciones.rb
module Excepciones extend self
  
  def get_usuario_no_valido
    return "El usuario no tiene autorización para modificar el/los recursos solicitados."
  end

  def get_conexion_bd_caida
    return "Error al conectar con la base de datos de SiReMu."
  end

  def get_lista_no_encontrada
    return "La lista de reproduccion provista no se encuentra en la base de datos de SiReMu."
  end

  def get_error_interno
    return "Error interno del servidor de SiReMu."
  end

  def get_lista_registrada
    return "El usuario ya tiene una lista registrada con ese nombre."
  end

end 